src/TextTruncate/TextTruncate.tsx

```diff
@@ -0,0 +1,79 @@
+import * as classnames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+
+import {Button} from "../Button/Button";
+
+import "./TextTruncate.less";
+
+const propTypes = {
+  className: PropTypes.string,
+  text: PropTypes.string.isRequired,
+  more: PropTypes.string,
+  less: PropTypes.string,
```

Comment:
nit: let's be more explicit with these prop names, so it's obvious what they do: like `showMoreLabel` and `showLessLabel` or something